al-cwe-collector.js

```diff
@@ -0,0 +1,109 @@
+'use strict';
+const AlAwsCollector = require('@alertlogic/al-aws-collector-js').AlAwsCollector;
+const m_packageJson = require('./package.json');
+const parse = require('@alertlogic/al-collector-js').Parse
+const async = require('async');
+
+const typeIdPaths = [
+    { path: ['detail', 'type'] }
+];
+
+const tsPaths = [
+    { path: ['time'] }
+];
+
+class CweCollector extends AlAwsCollector {
+    constructor(context, aimsCreds, formatMessages, healthChecks = [], statsChecks = []) {
+        super(context,
+            "cwe",
+            AlAwsCollector.IngestTypes.SECMSGS,
+            m_packageJson.version,
+            aimsCreds,
+            formatMessages,
+            healthChecks,
+            statsChecks);
+        this.stack_name = process.env.stack_name;
+    }
+
+    getProperties(event) {
+        const baseProps = super.getProperties();
+        const stack_name = event && event.ResourceProperties.StackName ? event.ResourceProperties.StackName : this.stack_name;
+        const collectRule = event && event.ResourceProperties.CollectRule ? event.ResourceProperties.CollectRule : `aws.guardduty`;
+
+        let cweProps = {
+            cf_stack_name: stack_name,
+            collect_rule: collectRule
+        };
+        return Object.assign(cweProps, baseProps);
+    };
+
+    register(event, custom, callback) {
+        let collector = this;
+        let cweRegisterProps = this.getProperties(event);
+        AlAwsCollector.prototype.register.call(collector, event, cweRegisterProps, callback);
+    }
+
+    process(event, callback) {
+        const context = this._invokeContext;
+        var collector = this;
+        async.waterfall([
+            function (asyncCallback) {
+                collector._formatFun(event, context, asyncCallback);
```

Comment:
let's modify [`formatFun`](https://github.com/alertlogic/cwe-collector/pull/66/files#diff-e727e4bdf3657fd1d798edcd6b099d6e092f8573cba266154583a746bba0f346R85) to return a json object.

Reply:
Removed JSON stringify from format function and return original  json object.